docs: Add Discover session panel creation from dashboard Add menu (#5974)
elastic/docs-content: explore-analyze/visualize.md

| --- | --- | --- |
| Visualizations | [Lens](visualize/lens.md) | Create powerful charts in {{kib}}. This is the default editor. |
|  | [{{esql}}](/explore-analyze/query-filter/languages/esql-kibana.md) | Create visualizations from {{esql}} queries |

> |  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a Discover session session directly in your dashboard without saving it to the library |

Reviewer: ```suggestion
|  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a Discover session directly in your dashboard without saving it to the library |
```

> |  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a [Discover session](../discover.md) session directly in your dashboard without saving it to the library |

Reviewer: ```suggestion
|  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a Discover session session directly in your dashboard without saving it to the library |
```

|  | [Maps](visualize/maps.md) | Create beautiful displays of your geographical data |
|  | [Alerts](visualize/alert-panels.md) | View Observability or Security alerts in your dashboard |
|  | [Custom visualizations](visualize/custom-visualizations-with-vega.md) | Use Vega to create new types of visualizations |

> |  | [Discover sessions](discover.md) | Explore your data using Discover and add the session as a panel directly in your dashboard |

Reviewer: ```suggestion
|  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a [Discover session](../discover.md) session directly in your dashboard without saving it to the library |
```

> |  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a [Discover session](../discover.md) session directly in your dashboard without saving it to the library |

Reviewer: ```suggestion
|  | [Discover sessions](discover.md) {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` | Create a [Discover session](../discover.md) session directly in your dashboard without saving it to the library |
|  | [Maps](visualize/maps.md) | Create beautiful displays of your geographical data |
|  | [Alerts](visualize/alert-panels.md) | View Observability or Security alerts in your dashboard |
|  | [Custom visualizations](visualize/custom-visualizations-with-vega.md) | Use Vega to create new types of visualizations |
```
Correct ordering in UI
